source/agora/utils/Test.d

```diff
@@ -540,8 +540,9 @@ public struct TxBuilder
 
     /***************************************************************************
 
-        Takes a block object and filters the spendable outputs
+        Takes a block object and filters the payment outputs
         into a `TxBuilder` object.
+        N.B. When calling this in your test be aware that not all of the payment outputs may still be available to spend (i.e. some may no longer be UTXOs). So consider what other transactions have been previousy made from this block's outputs.
```

Comment:
Add a newline after L544 to make the added documentation a  description ([see spec](https://dlang.org/spec/ddoc.html)).

Also, instead of `N.B.`, a `Note` section  could be used. But in this case, just using a description and losing the `N.B.` is enough. The description could IMO be simplified (a la "The outputs may not be spendable anymore if other blocks have been externalized after this `block`), but if you wish to keep it like this, please add hard breaks at 80 columns.